docs/product-assembly_dm-on-openshift/src/main/asciidoc/secrets-central-create-proc.adoc

```diff
@@ -8,23 +8,57 @@ If you are planning to deploy {CENTRAL} in your OpenShift environment, you must
 +
 See https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.1/html-single/Security_Guide/index.html#Generate_a_SSL_Encryption_Key_and_Certificate[Generate a SSL Encryption Key and Certificate] for more information on how to create a keystore with self-signed or purchased SSL certificates.
 +
-. Use the `oc` command to generate a secret named `decisioncentral-app-secret` from the new keystore file:
+. Use the `oc` command to generate a secret named 
```

Comment:
Just fyi, you can actually just say "named `{CENTRAL}-service-account`" Then no need for the ifdefs. Doc attributes work in unicode like this, although they don't work in code blocks.

Speaking of code blocks and ifdefs, I'm not sure what's going on in the blocks and ifdefs that follow, but the output is skewed. Something is missing or wrongly ifdefed. Check the built output and see what you think.

Reply:
{CENTRAL} expands to "Business Central" and I need "businesscentral"?

Reply:
@mramendi, oops, sorry, I meant the URL version of that attribute that I created a while back, which expands to `business-central` and `decision-central` accordingly. The variable is {URL_COMPONENT_CENTRAL} and is in the doc-attributes-pam and doc-attributes-dm adoc files. It's a lengthy attribute, but with how much it comes up in API calls, file names, etc, I found it very useful and very worthwhile to avoid all the ifdefs.

So this would be `{URL_COMPONENT_CENTRAL}-service-account`. Anyway, your call. Just a thought.